src/main/scala/edu/ie3/simona/event/ResultEvent.scala

```diff
@@ -15,7 +15,7 @@ import edu.ie3.datamodel.models.result.connector.{
 import edu.ie3.datamodel.models.result.system.SystemParticipantResult
 import edu.ie3.simona.agent.grid.GridResultsSupport.PartialTransformer3wResult
 
-sealed trait ResultEvent extends Event
+trait ResultEvent extends Event
```

Comment:
I think it's not that great to have all messages directly extend `ResultEvent`. 

One way to do it would to use [adapters](https://doc.akka.io/docs/akka/current/typed/interaction-patterns.html#adapted-response), but that has the drawback here that we have to send back the adapter's ActorRef to SimonaSim, and we want to keep the ResultEventListener as simple as possible.

So a better way to do this (I think) would be to have a trait `ResultMessage` and have `ResultEvent` extend that trait, as well as having ResultEventListener-specific messages (`SinkResponse` etc.) to extend `ResultMessage` as well.